.github/workflows/package-apisix-rpm-el7-with-apisix-openresty.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       - name: packaging APISIX
         run: |
           make package type=rpm app=apisix version=2.7 checkout=release/2.7 image_base=centos image_tag=7 openresty=apisix-openresty
+          make package type=rpm app=apisix version=2.9 checkout=release/2.9 image_base=centos image_tag=7 openresty=apisix-openresty
```

Comment:
Why use 2.9?

Reply:
@spacewander in order to put the rpm of the latest apisix to yum repository. What do you think? Thanks.

Reply:
@spacewander  After reconsidering this, I prefer to discard the adding of `2.9`. PR updated. Thanks for reminding.